src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentsLoadPageData.java

```diff
@@ -229,9 +229,9 @@ private FeedbackResponseCommentRow buildFeedbackResponseCommentAdd(FeedbackQuest
 
     private String getExtraClass(String giverEmail, String instructorEmail, boolean isPublic) {
         
-        return " giver_display-by-"
+        return " giver-display-by-"
```

Comment:
Since we're touching this code, maybe we could change it to 
```
return (giverEmail.equals(instructorEmail) ? "giver-display-by-you" : "giver-display-by-others")
        + (isPublic ? "status-display-public" : "status-display-private");
```
so that it is easier to detect/locate this class using search